src/sgame/sg_bot_skilltree.cpp

```diff
@@ -104,29 +143,31 @@ static void G_SetSkillsetBudgetAliens( int val )
 void G_InitSkilltreeCvars()
 {
 	static Cvar::Callback<Cvar::Cvar<std::string>> g_disabledSkillset(
-		"g_disabledSkillset",
-		"Disabled skills for bots, example: " QQ("mantis-attack-jump, prefer-armor"),
+		"g_bot_disabledSkillset",
```

Comment:
See discussion here about naming: https://github.com/Unvanquished/Unvanquished/pull/2697#discussion_r1225614467

I'm not too happy that with the current naming scheme in this PR the cvars which control the skill tree are spread around and not in the same namespace. I fear it may make it less discoverable / easy to tweak. (Maybe let's keep the discussion about the naming scheme on this thread (to avoid duplication): https://github.com/Unvanquished/Unvanquished/pull/2697#discussion_r1225614467

Reply:
I changed it to the `g_bot_skillset_` prefix.